Solitaire.java

```diff
@@ -113,10 +113,11 @@ public static boolean playAgain(Scanner s){
 	}
 
 	public static void encrypt(String e){
-		String original = e;
-		
 		//remove non-letters
-		//capitalize and remove spaces
+		String original = e.toUpperCase;
```

Comment:
I think the e.toUpperCase; should be e.toUpperCase();

Reply:
Yeah you're right.